### ISSUE TITLE
Relationships with Hiring Firms

### ISSUE THREAD

**Issue by meganmcgeee** [LearnTeachCode/leadership#21]
One of the big benefits of the bootcamp is assistance with job/apprenticeships/internship. Relationships with dev shops, start-ups, accelerators, co-working spaces, trips to job fairs and networking events.

**Comment by ghost** [comment]
yes. I'd like to help develop this too. General goal-setting and 'landscape painting' of technology/code.

**Comment by TheBeege** [comment]
To make sure I understand, are we saying we want to have people who specialize in teaching those soft skills? Are we saying we want to have individuals within L2C that have relationships with these companies for purposes of hiring? Just looking for clarity

**Comment by LearningNerd** [comment]
Yeah I'm a bit unclear on this one too. Can you rephrase it as an action, like "Reach out to and develop relationships with hiring firms" and add some more thoughts here when you have a chance? I do agree though, this is one of the big draws of bootcamps and if one day we actually try to create our own we would absolutely need relationships with hiring managers at some point if we were to compete with existing bootcamps. One day!

**Comment by meganmcgeee** [comment]
**Reach out to local dev shops/develop relationships with hiring firms.**

My experience with those who are very interested applying to bootcamps is that they tend to be prioritizing job acquisition/security. If the LTC-LA  is to become a bootcamp, maybe creating opportunities for members of the community to interview with dev shops/start-ups would be useful.

Even if that is just having a job resource list or informing people via slack/meet-up/facebook of job fair opportunities. 

I'm def interested in compiling/coding up such a thing- if that is of interest. :-)

**Comment by armaneous** [comment]
I may be of some use here, maybe. If someone wants to come up with a list of tech companies/firms, I can try to see what network I may have into them (yay, LinkedIn may actually be useful for once).

**Comment by TheBeege** [comment]
@x2adrew 
I spoke with Django Girls Seoul about sponsors. They very strongly recommended Microsoft. 

After Googling around, I keep seeing these companies mentioned in multiple locations with regards to education funding:
* Microsoft
* Google
* Hewlett Packard
* Amazon
* Intel
* Apple
* IBM

**Comment by TheBeege** [comment]
Oh, O'Reilly also provides sponsorship in the form of free books
